testnets/terpnettestnet/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "chain_name": "terpnettestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Terp-Net",
+  "chain_id": "athena-1",
+  "bech32_prefix": "terp",
+  "daemon_name": "terp",
+  "node_home": "$HOME/.terp",
+  
+  "codebase": {
+    "git_repo": "github.com/terpnetwork/terp-core",
+    "recommended_version": "v0.1.2",
+    "tendermint_version": "0.34.22",
+
+    "compatible_versions": [
+      "v0.1.2"
+    ],
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/terpnetwork/test-net/master/athena-2/genesis.json"
+    },
+    "binaries": {
+      "linux/amd64": "https://github.com/terpnetwork/terp-core/archive/refs/tags/v0.1.2.tar.gz"
+    }
+  },
+  "peers": {
+    "seeds": [
+      
+      {
+        "id": "9b9ff10893b11aa5588170b289222d4bb6785d97",
+        "address": "23.81.114.228:26656"
+      }
+    ],
+    "persistent_peers": [
+      
```

Comment:
got a few of these random extra blank lines that doesn't really need to be there... not a big deal but it'd be cleanest to not be there